### PR TITLE
New getting started guide

### PR DESCRIPTION
**Pull request by perkinsjr**

### CODE REVIEW

File: content/guides/tina-cloud/getting-started/deployment.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Tina Cloud and Deployment
 last_edited: '2021-07-19T15:36:36.046Z'
 ---
 
-## Connect to Tina Cloud 
+## Connect to Tina Cloud
```
Comment (DirtyF) [inline content/guides/tina-cloud/getting-started/deployment.md:6]: so do we call it Tina CMS now?

Comment (perkinsjr) [reply content/guides/tina-cloud/getting-started/deployment.md:6]: No it is still Cloud.

**Comment by logan-anderson** [comment]
Looking good! I think we need to publish a new version of the CLI and test it before this is merged.

**Comment by perkinsjr** [comment]
> Looking good! I think we need to publish a new version of the CLI and test it before this is merged.

What is the timeline of this @logan-anderson >?

**Comment by jeffsee55** [comment]
@perkinsjr I'm not sure why Logan's install was getting the old version, I tested it the other day. Can you give it a shot?

**Review by jeffsee55 — APPROVED** [review]

**Comment by logan-anderson** [comment]
@jeffsee55 @perkinsjr it looks like vercel didnt deploy this with the file size issue. https://vercel.com/tinacms/tinacms-site-next/5dhhLv5Nt83MsBniPz6PwAudVQQF

**Pull request merged**